napari/layers/labels/labels.py

```diff
@@ -547,9 +552,8 @@ def on_mouse_move(self, event):
                 interp_coord = interpolate_coordinates(
                     self._last_cursor_coord, self.coordinates, self.brush_size
                 )
-            with self.events.set_data.blocker():
-                for c in interp_coord:
-                    self.paint(c, new_label)
+            for c in interp_coord:
+                self.paint(c, new_label, refresh=False)
```

Comment:
@sofroniewn I'm having a bit of trouble interpreting the codecov UI, but, if I'm not mistaken, it shows that this loop is not covered by any tests:

https://codecov.io/gh/napari/napari/compare/6c5bb037e191bce3c05c5bf23f4f143ecc89a240...89db15b0491fc3f0322a4541eba773be441c12d1/diff

Could you verify that?

Reply:
that is likely correct, as it is inside our mouse callback. I can try adding test + benchmark before we merge